test/expected/validation.out

```diff
@@ -179,25 +179,25 @@ FROM (
      0
 (1 row)
 
-SELECT COUNT(*), COUNT(x.city), COUNT(DISTINCT x.id)
+SELECT COUNT(*), COUNT(x.city), COUNT(DISTINCT x.city)
```

Comment:
I'm not exactly sure, how much these 3 tests were specifically about operations on an AID column. If they don't make sense for non-aid column, please let me know

Reply:
Why is this rejected? Aggregating AID columns should be allowed. Same thing below.

Reply:
it SELECTs * from a personal relation, therefore including `id` - the AID

Reply:
You're right, stupid question on my part... I completely missed the `*` select in the subquery while looking at it.